src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -274,16 +283,18 @@ struct Mass : db::SimpleTag {
 };
 
 /*!
- * \brief Whether to apply the acceleration due to the scalar self-force to the
- * particle
+ * \brief The maximum number of iterations that will be applied to the
+ * acceleration of the particle.
  */
-struct ApplySelfForce : db::SimpleTag {
-  using type = bool;
-  using option_tags = tmpl::list<OptionTags::ApplySelfForce>;
+struct MaxIterations : db::SimpleTag {
```

Comment:
With this change, you now implicitly convert a `size_t` to a `bool` wherever you had a check for `ApplySelfForce`. Could you change those to explicitly check if the `size_t > 0`? Also, depending on your thoughts about my other comment, this may be moot.